src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -389,6 +388,21 @@
             :render-data               text-input-ref
             :render-fn                 mention-item}]]]))))
 
+(defn autocomplete-mentions [suggestions]
+  [:f>
```

Comment:
this file is huge. Is there a clean way to break it up?

Reply:
Thank you for your reviews @J-Son89 
There are multiple files that I would like to break up, but it will cause circular dependencies.
I think it will get smaller when we refactor the old code.